adapters/alkimi/alkimi.go

```diff
@@ -0,0 +1,157 @@
+package alkimi
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/floors"
```

Comment:
@kalidas-alkimi, prebid server has published major release 2.0. Requesting to sync with `Alkimi-Exchange:master` with latest prebid server master.

Reply:
fixed